Makefile

```diff
@@ -6,7 +6,7 @@ help:
 	@echo "  sensor        run sensor space processing"
 	@echo "  source        run source space processing"
 	@echo "  profile       to profile memory consumption"
-	@echo "  all           fetch data and run full pipeline
+	@echo "  all           fetch data and run full pipeline"
```

Comment:
The options in the Makefile should be documented in the main README

Reply:
see the Makefile section under Usage in https://github.com/mne-tools/mne-study-template/pull/35/commits/9527de915ac0ea48ecadedfa3f9a5e58a2240332 is that sufficient?